Corona-Warn-App/src/main/java/de/rki/coronawarnapp/appconfig/mapping/CWAConfigMapper.kt

```diff
@@ -33,34 +34,15 @@ class CWAConfigMapper @Inject constructor() : CWAConfig.Mapper {
             else -> supportedCountriesList
         }
 
-    private fun ApplicationConfigurationAndroid.isDeviceTimeCheckDisabled(): Boolean {
-        if (!hasAppFeatures()) return false
+    private fun ApplicationConfigurationAndroid.isDeviceTimeCheckDisabled() = findBoolean(
+        labelValue = "disable-device-time-check",
+        defaultValue = false
+    )
 
-        return try {
-            (0 until appFeatures.appFeaturesCount)
-                .map { appFeatures.getAppFeatures(it) }
-                .firstOrNull { it.label == "disable-device-time-check" }
-                ?.let { it.value == 1 }
-                ?: false
-        } catch (e: Exception) {
-            Timber.e(e, "Failed to map `disable-device-time-check` from %s", this)
-            false
-        }
-    }
-
-    private fun ApplicationConfigurationAndroid.isUnencryptedCheckInsEnabled(): Boolean {
-        if (!hasAppFeatures()) return false
-        return try {
-            (0 until appFeatures.appFeaturesCount)
-                .map { appFeatures.getAppFeatures(it) }
-                .firstOrNull { it.label == "unencrypted-checkins-enabled" }
-                ?.let { it.value == 1 }
-                ?: false
-        } catch (e: Exception) {
-            Timber.e(e, "Failed to map `unencrypted-checkins-enabled` from %s", this)
-            false
-        }
-    }
```

Comment:
Created the new extension function `findBoolean()` to avoid duplicated code like this.